src/java.desktop/share/classes/java/awt/font/TextMeasurer.java

```diff
@@ -247,8 +243,7 @@ private void initAll(AttributedCharacterIterator text) {
                 GraphicAttribute graphic = (GraphicAttribute)
                                 paragraphAttrs.get(TextAttribute.CHAR_REPLACEMENT);
                 fBaseline = TextLayout.getBaselineFromGraphic(graphic);
-                Hashtable<Attribute, ?> fmap = new Hashtable<>(5, (float)0.9);
-                Font dummyFont = new Font(fmap);
+                Font dummyFont = new Font(Collections.emptyMap());
```

Comment:
Is this change correct taking into account immutability of `Collections.emptyMap()`?

Reply:
I believe yes. Constructor `Font(Map<? extends Attribute, ?> attributes)` doesn't modify its parameter.

Reply:
I guess the null will work as well here?

Reply:
`null` works. Updated.